mesoSPIM/src/mesoSPIM_ImageWriter.py

```diff
@@ -130,10 +195,32 @@ def prepare_acquisition(self, acq, acq_list):
             self.mip_image = np.zeros((self.y_pixels, self.x_pixels), 'uint16')
 
         self.cur_image = 0
```

Comment:
probably worth renaming `cur_image` ->  `cur_image_counter` (integer) to avoid confusion with `image` variable (an array).